proj2.exs

```diff
@@ -0,0 +1,12 @@
+usage_string = "Invalid arguments to the command\nUsage: mix run proj2.exs --no-halt <numNodes:int> <topology:string> <algorithm:string>"
+try do
+  if length(System.argv) != 3 do
+    IO.puts "--------- #{length(System.argv)}"
+    raise ArgumentError
```

Comment:
Just raising an error with a message should work right ? https://hexdocs.pm/elixir/Kernel.html#raise/1